doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,140 @@
+ * The external memory must be large enough to store the firmware patches, taking into consideration also the patch upgrade requirement, for example, Device Firmware Update (DFU).
+   The maximum size of all the firmware patches combined is 128 KB.
+
+.. note::
+    Currently, in Nordic Semiconductor SoCs, access to external memory and the Execute in Place (XIP) feature are available through the QSPI interface only.
+
+Supported platforms
+===================
+
+The following platforms are supported:
+
+* nRF5340 SoC or nRF5340 DK with nRF7002 EK as a shield
+* nRF52840 SoC or nRF52840 DK with nRF7002 EK as a shield
+
+.. note::
+    Due to limitations mentioned above, the nRF7002 DK is not supported as it needs access to external memory through the SPI interface.
+
+Available options
+*****************
+
+There are two ways to use the firmware patches in the external memory feature:
+
+* XIP supported external memory
+* Non-XIP supported external memory
+
+
+XIP supported external memory
+=============================
+
+If the application supports XIP from external memory, then the firmware patches can be loaded as part of the nRF Wi-Fi driver code and then relocated to the external memory and executed from there.
+To enable this feature, set the :kconfig:option:`CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP` kconfig option to ``y``.
+Once the build is complete, the feature can be verified by checking the memory regions summary in the build output.
+A new memory region called ``EXTFLASH`` is added to the memory regions summary, and the firmware patches are placed in this region.
+The size of the ``FLASH`` used is reduced by the size of the firmware patches.
+
+A sample memory regions summary is shown below:
+
+.. code-block:: console
+
+  Memory region         Used Size  Region Size  %age Used
+        EXTFLASH:       64544 B         8 MB      0.77%
+           FLASH:      618752 B         1 MB     59.01%
+             RAM:      170636 B       448 KB     37.20%
+        IDT_LIST:          0 GB         2 KB      0.00%
+
+.. note::
+    Depending on XIP implementation, this can cause a performance impact in both time and power.
+
+non-XIP Supported external memory
+=================================
+
+If the application does not support XIP from external memory, then the firmware patches have to be copied to the internal flash and downloaded to the nRF70 device.
+This feature can eb enabled using Zephyr's configuration :ref:`snippets` feature.
+
+
+Configuration
+-------------
+
+The following configuration options are available:
+
+* :kconfig:option:`CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_STORE`: Enables the feature.
+* :kconfig:option:`CONFIG_NRF_WIFI_FW_FLASH_CHUNK_SIZE`: Defines the size of the chunks used to write the firmware patches to the external memory.
+  The default value is 8192 bytes.
+
+* ``nrf70_fw_partition`` : Defines the name of the external memory partition that stores the firmware patches.
```

Comment:
MAybe add a more direct guideline, sayng that the user must have configure a DTS partiion that instructs the Wi-Fi driver on how to access the patch.